js/minirepl.js

```diff
@@ -109,7 +109,8 @@ function compileCode(sourceEditor, targetEditor) {
 
   try {
     transformed = Babel.transform(sourceEditor.getValue(), {
-      presets: ["es2015", "es2016", "es2017"],
+      presets: ["react", ["env", { targets: "defaults", loose: true }]],
```

Comment:
I suppose defaults still compiles basically everything? Maybe in the future we want to make targets a simple dropdown even for the landing page 🤔

Reply:
FYI here is what will be compiled on "defaults": https://github.com/babel/babel/blob/master/packages/babel-preset-env/test/fixtures/debug/browserslists-defaults/stdout.txt

And `"defaults not ie": https://github.com/babel/babel/blob/master/packages/babel-preset-env/test/fixtures/debug/browserslists-defaults-not-ie/stdout.txt

I am good to remove `ie` here, WDYT?

Reply:
Yeah, I'm fine with dropping:

![image](https://user-images.githubusercontent.com/56288/78845952-b8922c80-79cf-11ea-8f96-523f42128d63.png)